sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java

```diff
@@ -41,10 +46,146 @@ public CreateNewEventSteps(WebDriverHelpers webDriverHelpers) {
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+        });
+    When(
+        "^I validate create a new event popup",
+        () -> {
+          String timestamp = String.valueOf(System.currentTimeMillis());
+          webDriverHelpers.fillInWebElement(TITLE_INPUT, "EVENT_AUTOMATION_" + timestamp);
+          selectEventStatus("SIGNAL");
+          selectEventStatus("EVENT");
+          selectEventStatus("SCREENING");
+          selectEventStatus("CLUSTER");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(PRIMARY_MODE_OF_TRANSMISSION);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NOSOCOMIAL);
+          selectEventStatus("DROPPED");
+          selectEventManagementStatusOption("PENDING");
+          selectEventManagementStatusOption("ONGOING");
+          selectEventManagementStatusOption("DONE");
+          selectEventManagementStatusOption("CLOSED");
+          selectRiskLevel("Low risk");
+          selectRiskLevel("Moderate risk");
+          selectRiskLevel("High risk");
+          selectRiskLevel("Unknown");
+          webDriverHelpers.scrollToElement(DISEASE_INPUT);
+          selectEventInvestigationStatusOptions("INVESTIGATION PENDING");
+          selectEventInvestigationStatusOptions("ONGOING INVESTIGATION");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectEventInvestigationStatusOptions("INVESTIGATION DONE");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectEventInvestigationStatusOptions("INVESTIGATION DISCARDED");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_START_DATE);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_INVESTIGATION_END_DATE);
+          selectDisease("Acute Flaccid Paralysis");
+          selectDisease("Anthrax");
+          selectDisease("COVID-19");
+          selectDisease("Cholera");
+          selectDisease("Congenital Rubella");
+          selectDisease("Dengue Fever");
+          selectDisease("Ebola Virus Disease");
+          selectDisease("Guinea Worm");
+          selectDisease("Human Rabies");
+          selectDisease("Influenza (New subtype)");
+          selectDisease("Lassa");
+          selectDisease("Measles");
+          selectDisease("Meningitis (CSM)");
+          selectDisease("Monkeypox");
+          selectDisease("Not Yet Defined");
+          selectDisease("Other Epidemic Disease");
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DISEASE_NAME);
+          selectDisease("Plague");
+          selectDisease("Poliomyelitis");
+          selectDisease("Unspecified VHF");
+          selectDisease("Yellow Fever");
```

Comment:
@BojteTamas  should we override this method to give a list as parameter and select them in a for each?

Reply:
We need to remove the logic from here because we are doing too much here.